### DISCUSSION TITLE
Webpack not working ...

### DISCUSSION THREAD

**Discussion by cavemansspa** [MithrilJS/mithril.js#2712] [Q&A]
The current website documentation regarding Webpack / Mithril setup is outdated.  See the gitter links below for latest recommended Webpack setup.

* https://gitter.im/mithriljs/mithril.js?at=61afab5acdb5c1081a4fad13
* https://gitter.im/mithriljs/mithril.js?at=61afd26dc2cc0e5343ed5141

**Comment by sho88** [comment]
In the package.json file, you can add a "start" entry to the scripts section in `package.json`.

```
{
  // ...
  "scripts": {
    "webpack:watch": "webpack -w --mode 'development' -o './build' --output-filename 'bundle.min.js' --performance-hints 'warning',
  }
}
```

"development" mode parameter in that script is useful when working locally, however, when working on production, it is strongly recommended to change that to "production". In fact, for production, it would be better to have a `webpack.config.js` file and make the necessary changes in there.
